SIAB/spillage/api.py

```
@@ -33,157 +35,351 @@ def _coef_gen(rcut: float, ecut: float, lmax: int, value: str = "eye"):
     import os
 
-    jy = "orb_mat" not in init_recipe
+    jy = "orb_mat" not in guess
     initgen = initgen_jy if jy else initgen_pw
     if jy:
-        nzeta0 = [0] * len(nzeta) if nzeta0 is None else nzeta0
-        ib = _nzeta_analysis(os.path.dirname(init_recipe["outdir"])) # indexed by [ispin][l] -> list of band index
-        ib = [[ibsp[l][(2*l+1)*nz0:(2*l+1)*nz] for l, (nz, nz0) in enumerate(zip(nzeta, nzeta0))]
+        excluded = [0] * len(nzeta) if excluded is None else excluded
+        ib = _nzeta_analysis(os.path.dirname(guess["outdir"])) # indexed by [ispin][l] -> list of band index
+        ib = [[ibsp[l][(2*l+1)*nz0:(2*l+1)*nz] for l, (nz, nz0) in enumerate(zip(nzeta, excluded))]
```

Ruff

Ambiguous variable name: `l`

(E741)